modules/videoio/src/cap_avfoundation_mac.mm

```diff
@@ -293,7 +293,7 @@ - (IplImage*)getOutput;
 
     [mCaptureSession release];
     [mCaptureDeviceInput release];
-    [mCaptureDevice release];
+    // [mCaptureDevice release]; fix #7833
```

Comment:
It is not fixed yet.
I'm asking to replace this line to `mCaptureDevice = nil;` and to make similar change in file "cap_avfoundation.mm".

BTW, Are you able to reproduce this problem on your side (and to validate fix)?